src/main/java/org/gitlab4j/api/models/MergeRequest.java

```diff
@@ -86,6 +87,14 @@ public void setApprovalsBeforeMerge(Integer approvalsBeforeMerge) {
         this.approvalsBeforeMerge = approvalsBeforeMerge;
     }
 
+    public List<Assignee> getAssignees() {
+        return assignees;
+    }
+
+    public void setAssignees(List<Assignee> assignees) {
+        this.assignees = assignees;
+    }
+
```

Comment:
@eedzjee These should be moved to after ```setAssignee(Assignee assignee)```

Reply:
You're right! I've updated the file.

Reply:
Thanks for making the change.   The next release is scheduled for 2/10/2020, will let you know when it is released.